peps/pep-0784.rst

```diff
@@ -273,8 +254,8 @@ How to Teach This
 =================
 
 Documentation for the new module is in the reference implementation branch. The
-documentation for other modules will be updated to discuss the deprecation of
-their existing import names, and how to migrate.
+documentation for existing modules will be updated to reference the new names
+as well.
```

Comment:
Do the new names now become canonical? The PEP should take a position.

Reply:
I think yes the new names should become canonical.  Let's see what other SC members think.

Reply:
The intent is for them to become canonical as that is why the compression namespaces is useful.  I expect we may iterate on how exactly to present this in our docs outside the PEP. 

As far as the text in the PEP goes adding something along the lines of:

"The use of the new compression names will be promoted over the original top level module names in the Python documentation when the users minimum Python version support requirements makes that feasible."

Reply:
I added a sentence about this in 53ab0b58. Not sure if I should be stronger in my wording but hopefully that clearly makes a stance.

Reply:
Oh, my page was open from last night and I missed Greg's comment. I'll include that text.

Reply:
Added in 9d58ebce, I made a slight edit to the sentence to clarify, hopefully that's ok!